### ISSUE TITLE
Using `z.instanceOf()` to reference a class type

### ISSUE THREAD

**Issue by kvithana** [sachinraja/zod-to-ts#7]
Hi there! Thanks for this cool library. I'm trying to use it as a codegen tool to generate types based on BigQuery schema definitions and output them to a file.

I have a use case such as this:

```
const schema = z.object({
  name: z.string(),
  type: z.instanceof(BigQueryTimestamp),
})

const { node } = zodToTs(schema)
console.log(printNode(node))
```

Which results in a generated string of:
```
{
    name: string,
    type: any
}
```

What I would like is something which results in a generated string of:
```
{
    name: string,
    type: BigQueryTimestamp
}
```

And then I could manually add `import { BigQueryTimestamp } from '@google-cloud/bigquery'` to the top of the output file.

Is there a way I specify the identifier for `type`? Or is this something to do with when it's printed? Not super familiar with how the underlying typescript stuff works but happy to dig into it and contribute if you can point me in the right direction. Thanks!

**Comment by sachinraja** [comment]
Hey, just published v0.2.1, which should allow you to do this. You can use `withGetType` to [override any type](https://github.com/sachinraja/zod-to-ts#overriding-types). So that would make your schema look like this:
```ts
import { z } from 'zod'
import { withGetType, zodToTs } from 'zod-to-ts'

const BigQueryTimestampSchema = withGetType(
  z.instanceof(BigQueryTimeStamp),
  (ts) => ts.factory.createIdentifier('BigQueryTimestamp'),
)

const ItemSchema = z.object({
  name: z.string(),
  type: BigQueryTimestampSchema,
})

const { node } = zodToTs(ItemSchema, 'Item')
```

**Comment by kvithana** [comment]
Oh, amazing! Thanks so much for the update.
